kuma/wiki/models.py

```diff
@@ -1743,11 +1743,8 @@ def previous(self):
         Returns the previous approved revision or None.
         """
         try:
-            return self.document.revisions.filter(
-                is_approved=True,
-                created__lt=self.created,
-            ).order_by('-created')[0]
-        except IndexError:
+            return self.get_previous_by_created(is_approved=True)
+        except self.DoesNotExist:
             return None
```

Comment:
Hmm, ok. I wonder if we can add a test that exposes this so if anyone tries this again the test fails?